.babelrc

```diff
@@ -1,10 +1,18 @@
 {
-    "presets": [
-      "@babel/preset-env",
-      "@babel/preset-react",
-      "@babel/preset-typescript"
-    ],
-    "plugins": [
-      "@babel/plugin-proposal-class-properties",
-    ],
-  }
\ No newline at end of file
+  "presets": [
+    "@babel/preset-env",
+    "@babel/preset-react",
+    "@babel/preset-typescript"
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties",
+    [
+      "i18next-extract",
```

Comment:
I thought we wanted to remove this? At least it was removed in some of the other repositories because it caused more harm than it did good.

@brandones knows more.

Reply:
Yeah in my experience it's extreeeeeemely finicky. More often than not randomly removes translations that are actually in use. I'd recommend against it @donaldkibet .